src/screens/Ticketing/Purchase/Overview/components/Summary.tsx

```diff
@@ -0,0 +1,91 @@
+import ThemeText from '@atb/components/text';
+import {ActivityIndicator, StyleProp, View, ViewStyle} from 'react-native';
+import Button from '@atb/components/button';
+import {PurchaseOverviewTexts, useTranslation} from '@atb/translations';
+import {ArrowRight} from '@atb/assets/svg/mono-icons/navigation';
+import React from 'react';
+import {StyleSheet} from '@atb/theme';
+import {formatDecimalNumber} from '@atb/utils/numbers';
+import {useNavigation} from '@react-navigation/native';
+import {OverviewNavigationProp} from '@atb/screens/Ticketing/Purchase/Overview';
+import {TariffZoneWithMetadata} from '@atb/screens/Ticketing/Purchase/TariffZones';
+import {UserProfileWithCount} from '@atb/screens/Ticketing/Purchase/Travellers/use-user-count-state';
+import {PreassignedFareProduct} from '@atb/reference-data/types';
+
+type Props = {
+  price: number;
+  isLoading: boolean;
+  isError: boolean;
+  fromTariffZone: TariffZoneWithMetadata;
+  toTariffZone: TariffZoneWithMetadata;
+  userProfilesWithCount: UserProfileWithCount[];
+  preassignedFareProduct: PreassignedFareProduct;
+  travelDate?: string;
```

Comment:
This might rise the question "Why not just pass the `toPaymentFunction` from the overview-screen? Then we don't need this many parameters".

My thought is that the Overview-components (Summary, ZoneItem, and others we will make) encapsulates the logic about where they are going to navigate. They are not (as of now) reusable components, so we don't need to concern the parent component about specifying where the component should navigate. For the summary component it makes it necessary to pass along quite a few parameters, but this will not be the case for the other components.